HaulageApplication/Haulage/DriverPages/AddTripExpensePage.xaml.cs

```diff
@@ -0,0 +1,48 @@
+using SQLite;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Haulage.DriverPages;
+
+public partial class AddTripExpensePage : ContentPage
+{
+    public AddTripExpensePage()
+    {
+        InitializeComponent();
+    }
+
+    private void SaveExpenseRecord_Clicked(object sender, EventArgs e)
+    {
+        Button SaveExpenseRecord = FindByName("SaveExpenseRecord") as Button;
+        SaveExpenseRecord.Clicked += SaveExpenseRecord_Clicked;
+
+        string expenseId = ExpenseIdEntry.Text;
+        string driverId = DriverIdEntry.Text;
+        string vehicleId = VehicleIdEntry.Text;
+        string fullname = FullnameEntry.Text;
+        string expense = ExpenseEntry.Text;
+        string expenseDescription = ExpenseDescEntry.Text;
+        string expenseCost = ExpenseCostEntry.Text;
+
+        var sql = $"INSERT INTO Expense VALUES ({expenseId}, {driverId}, {vehicleId}, '{fullname}', '{expense}', '{expenseDescription}', '{expenseCost}');";
```

Comment:
this functionality should be inside a databaseExecution method following the pattern perviously established.

Reply:
Due to time constraints I implemented this within the xaml as I was able to reuse code from earlier issues. In future development to produce cleaner code I would make this change to better adhere to software engineering principles